pages/formCountry.js

```diff
@@ -0,0 +1,5 @@
+import FormCountry from "@/components/FormCountryPage";
```

Comment:
![Bildschirm­foto 2023-08-16 um 11 16 11](https://github.com/KilianBau/capstone-project/assets/90191489/9acb127b-4654-4810-bc48-eb61afdd2624)

I usually treat -Form as a suffix rather than a prefix, such as `CountryForm`, because that's how you'd probably describe it in a conversation as well. "Can you fill out the form country" doesn't quite get the message across while "Can you fill out the country form"  does.

Reply:
Thanks for your review :)